src/Console/Command/FixCommand.php

```diff
@@ -309,6 +315,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $verbosity = $output->getVerbosity();
         $resolver = new ConfigurationResolver();
+        $resolver->getReportFactory()
+            ->setAddAppliedFixers(OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity())
+            ->setIsDecoratedOutput($output->isDecorated())
+            ->setStopwatch($this->stopwatch)
```

Comment:
:+1: for removing stopwatch dependency from Report.

But doing that please remember about:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1824/files#r56658635
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1824/files#r56659028